fsw/inc/cs_msg.h

```diff
@@ -48,8 +48,8 @@ typedef struct
     uint8   TablesCSState;               /**< \brief CS Tables table checksum state */
     uint8   OSCSState;                   /**< \brief OS code segment checksum state */
     uint8   CfeCoreCSState;              /**< \brief cFE Core code segment checksum stat e*/
-    uint8   RecomputeInProgress;         /**< \brief CS "Recompute In Progress" flag */
-    uint8   OneShotInProgress;           /**< \brief CS "OneShot In Progress" flag */
+    bool    RecomputeInProgress;         /**< \brief CS "Recompute In Progress" flag */
+    bool    OneShotInProgress;           /**< \brief CS "OneShot In Progress" flag */
```

Comment:
Things in "msg.h" affect inter-processor interfaces, so these should be limited should stay explicit about the width (i.e. keep using `uint8` here)